chart/templates/rbac/security-context-constraint-rolebinding.yaml

```
@@ -0,0 +1,93 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+################################
+## Airflow SCC Role Binding
+#################################
+{{- if and .Values.rbac.create .Values.rbac.createSCCRoleBinding }}
+{{- $schedulerLaunchExecutors := list "LocalExecutor" "CeleryExecutor" "KubernetesExecutor" "CeleryKubernetesExecutor" }}
+{{- $workerLaunchExecutors := list "CeleryExecutor" "KubernetesExecutor" "CeleryKubernetesExecutor" }}
```

In this context, I'm not sure we care if the scheduler or workers can launch pods, only if the component exists? Obviously we need a scheduler, so no conditional there. Maybe this is what we want instead for the workers:

```suggestion
{{- $hasWorkers := has .Values.executor (list "CeleryExecutor" "KubernetesExecutor" "CeleryKubernetesExecutor") }}
```

Then the worker serviceaccount section can be done conditionally based on `$hasWorkers`?

---

Yes, I agree, it does look much better like that.
As for schedulers, I think we can remove the test altogether since we always have one in place.